src/views/v2/list/nav/next.php

```diff
@@ -14,17 +14,22 @@
  * @version TBD
  *
  */
-if ( empty( $link ) ) {
-	return;
-}
 ?>
 <li class="tribe-common-c-nav__list-item">
-	<a
-		href="<?php echo esc_url( $link ); ?>"
-		rel="next"
-		class="tribe-common-c-nav__next"
-		data-js="tribe-events-view-link"
-	>
+	<?php if ( empty( $link ) ) : ?>
```

Comment:
Great stuff! I was thinking we could have a partial for the disabled ones and this could look a bit more clean. What do you think? I'm not a big fan of large if/else within templates, I have to say though.

Reply:
I'm OK either way, though I do lean a bit more toward the `if/else` in this case as there is a lot of repeated pieces. @bordoni @ryanurban thoughts?

Reply:
I feel like a case for each creates an easier overall template structure.

Reply:
sounds good, will separate

Reply:
I could also see either way being "better". @bordoni, given your institution knowledge here and with our customers and their feedback, have you noticed any sort of pattern around their thoughts and requests for ease of templateage overrides? Better to keep it all inside one template with conditionals to cut down on needing to nav around files vs. having things be cleaner in two templates but now needing to track two if doing customizations around the pagination?

Reply:
Normally customers tend to appreciate when the is less duplicated logic for the sake of HTML.